modules/core/04-channel/keeper/grpc_query_test.go

```diff
@@ -193,6 +197,7 @@ func (suite *KeeperTestSuite) TestQueryChannels() {
 			if tc.expPass {
 				suite.Require().NoError(err)
 				suite.Require().NotNil(res)
+				suite.Require().Equal(tc.expectedChannels, res.Pagination.Total)
 				suite.Require().Equal(expChannels, res.Channels)
 				suite.Require().Equal(len(expChannels), int(res.Pagination.Total))
```

Comment:
Sorry for slowing this down but I think this pretty much the same check as below too, right?
`suite.Require().Equal(len(expChannels), int(res.Pagination.Total))`

We can probably remove the `tc.expectedChannels`.

I think as long as the scenario from the original issue has been tested and covered then I'm happy to get this merged.

Reply:
Yeah, I agree with @damiannolan and I think we can remove `tc.expectedChannels`.